governance/roles.md

```diff
@@ -65,11 +65,8 @@ The general list of activities for TL are:
   * Propose agenda items for meetings to ensure that open issues are
   discussed with the group when needed
 
-TLs are elected by the following process:
-  1. One of the Chairs nominates a candidate to be a TL, and creates a PR that
-  adds the TL to the [SIG README](/README.md).
-  1. Members are invited to comment and vote according to the regular voting
-  process described below.
+TLs are assigned by CNCF Technical Oversight Committee
```

Comment:
Per the CNCF Special Interest Groups ("SIGs") charter, the TOC and Chairs nominate Tech leads.

Reply:
The key difference is that tech leads require a vote of the TOC (not the SIG membership or chair majority.  Maybe simply a reference without an attempt to summarize, because the TOC process isn't super clear about what happens before the vote (and I think there's a PR open on that)